operate/qa/integration-tests/src/test/java/io/camunda/operate/elasticsearch/FinishedImportingIT.java

```diff
@@ -0,0 +1,134 @@
+import io.camunda.operate.schema.indices.ImportPositionIndex;
+import io.camunda.operate.util.OperateZeebeAbstractIT;
+import io.camunda.operate.util.TestSupport;
+import io.camunda.operate.zeebeimport.ZeebeImporter;
+import io.camunda.operate.zeebeimport.elasticsearch.ElasticsearchRecordsReader;
+import io.camunda.zeebe.exporter.ElasticsearchExporter;
+import io.camunda.zeebe.exporter.ElasticsearchExporterConfiguration;
+import io.camunda.zeebe.exporter.test.ExporterTestConfiguration;
+import io.camunda.zeebe.exporter.test.ExporterTestContext;
+import io.camunda.zeebe.exporter.test.ExporterTestController;
+import io.camunda.zeebe.protocol.record.Record;
+import io.camunda.zeebe.protocol.record.RecordValue;
+import io.camunda.zeebe.protocol.record.ValueType;
+import io.camunda.zeebe.test.broker.protocol.ProtocolFactory;
+import java.io.IOException;
+import java.time.Duration;
+import java.util.Arrays;
+import org.awaitility.Awaitility;
+import org.elasticsearch.action.admin.indices.refresh.RefreshRequest;
+import org.elasticsearch.action.search.SearchRequest;
+import org.elasticsearch.client.RequestOptions;
+import org.elasticsearch.client.RestHighLevelClient;
+import org.elasticsearch.search.SearchHit;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class FinishedImportingIT extends OperateZeebeAbstractIT {
+  private static final ElasticsearchExporter EXPORTER = new ElasticsearchExporter();
+  private static final ElasticsearchExporterConfiguration CONFIG =
+      new ElasticsearchExporterConfiguration();
+  @Autowired public SchemaManager schemaManager;
+  @Autowired protected ZeebeImporter zeebeImporter;
+  @Autowired private RestHighLevelClient esClient;
+  @Autowired private OperateProperties operateProperties;
+  @Autowired private ImportPositionIndex importPositionType;
+  private final ProtocolFactory factory = new ProtocolFactory();
+
+  @Before
+  public void beforeEach() {
+    operateProperties.getImporter().setImportPositionUpdateInterval(1000);
+    CONFIG.index.prefix = operateProperties.getZeebeElasticsearch().getPrefix();
+    CONFIG.index.setNumberOfShards(1);
+    CONFIG.index.setNumberOfReplicas(0);
+    CONFIG.index.createTemplate = true;
+    CONFIG.retention.setEnabled(false);
+    CONFIG.bulk.size = 1; // force flushing on the first record
+
+    // here enable all indexes that needed during the tests beforehand as they will be created once
+    TestSupport.provideValueTypes()
+        .forEach(valueType -> TestSupport.setIndexingForValueType(CONFIG.index, valueType, true));
+
+    final var exporterTestContext =
+        new ExporterTestContext()
+            .setConfiguration(new ExporterTestConfiguration<>("elastic", CONFIG));
+    EXPORTER.configure(exporterTestContext);
+    EXPORTER.open(new ExporterTestController());
+  }
+
+  @Test
+  public void shouldMarkRecordReadersAsCompletedIf870RecordReceived() throws IOException {
```

Comment:
Can we also add tests for:

 * we write multiple different records/value types
 * plus multi partitions